rj_gameplay/rj_gameplay/play/basic_offense.py

```diff
@@ -79,6 +84,12 @@ def tick(
             if seek_tactic.is_done(world_state):
                 self._state = State.INIT_PASS
             """
+            if self._striker_st_time is None:
+                self._striker_st_time = time.time()
+            if (time.time() - self._striker_st_time) > 5:
+                print("striker failed, reassigning")
```

Comment:
not part of this pr ofc, but we should move this to ui or actual logging instead of just printing later.

Reply:
comment added